lib/floe/runner.rb

```diff
@@ -17,7 +17,7 @@ def register_scheme(scheme, klass_or_proc)
 
       private def resolve_scheme(scheme)
         runner = @runners[scheme]
-        runner = @runners[scheme] = @runners[scheme].call if runner.is_a?(Proc)
+        runner = @runners[scheme] = @runners[scheme].call if runner.kind_of?(Proc)
```

Comment:
this is a lazy eval, but only the first time hit.
I do wonder if this is not thread safe. Ruby's GPL probably takes care of it.